guides/session.rst

```diff
@@ -1,76 +1,114 @@
 Controlling the Browser
 =======================
 
-Ok. Now we know how to create the browser driver to talk with a specific
-browser emulator. Although we can use drivers directly to call some actions
-on the emulator, Mink provides a better way - ``Session``:
+In Mink, the entry point to the browser is called the session. Think about
+it as being your browser window (some drivers even allow to deal with switching
+tabs).
+
+The first thing to do with your session is to start it. Nothing can be done
+with it before starting it.
 
 .. code-block:: php
 
-    // init session:
+    // choose a Mink driver. More about it in later chapters
+    $driver = new \Behat\Mink\Driver\GoutteDriver();
+
     $session = new \Behat\Mink\Session($driver);
 
-    // start session:
+    // start the session
     $session->start();
 
 .. note::
 
-  As you can see, the first argument to the session (``$driver``) is just
-  a simple driver instance, which we created in the previous chapter.
+    The first argument to the session constructor is a driver object, which
+    is how the Mink abstraction layer works. You will discover more about
+    the available drivers in a :doc:`later chapter </guides/drivers>`.
```

Comment:
Hi guys!

Actually "a later chapter" is better now. I guess it's because "later chapter" is not referring to something specific. I would "available drivers in the chapter on Drivers".

Cheers!